client/web/src/user/profile/UserProfile.tsx

```diff
@@ -4,33 +4,54 @@ import { formatDistanceToNowStrict } from 'date-fns'
 
 import { UserAreaRouteContext } from '../area/UserArea'
 
+type userData =
```

Comment:
Typical convention to distinguish the type definition from the variable definition:

```suggestion
type UserData =
```